### PR TITLE
docs: Update UI text.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>

### What type of PR is this?

/kind documentation

### What this PR does / why we need it:

Update UI text.

### Which issue(s) this PR fixes:

Fixes ## [4076](https://github.com/kubesphere/kubesphere/issues/4076)

### Special notes for reviewers:

@harrisonliu5 The following files involve changes in the source code logic. Please take a look. Thanks!

src/components/Charts/Area/MediumArea/index.jsx

src/pages/workspaces/containers/Overview/ResourceUsage/Statistics/index.jsx

src/components/Modals/Delete/index.jsx

src/pages/workspaces/components/Modals/WorkspaceDelete/index.jsx

### Does this PR introduced a user-facing change?

```release-note
None
```

### CODE REVIEW

File: src/components/Modals/Delete/index.jsx [hunk 1]
```diff
@@ -111,7 +111,7 @@ export default class DeleteModal extends React.Component {
           <div className="h5">
             <Icon name="close" type="light" className={styles.closeIcon} />
             {title ||
-              (resource.split(', ').length === 1
+              (resource && resource.split(', ').length === 1
                 ? t('DELETE_TITLE_SI', { type: t(typeKey) })
                 : t('DELETE_TITLE_PL', { type: t(typeKeyPl) }))}
```
Comment (harrisonliu5) [inline src/components/Modals/Delete/index.jsx:116]: It is right that the title is 'DELETE_TITLE_PL' which is the resource is undefined or not the string?

**Comment by Patrick-LuoYu** [comment]
/assign @harrisonliu5 
Further updated. Please take a look. Thanks!
image

**Pull request merged**